packages/neo-one-editor/src/monaco/AsyncLanguageService.ts

```diff
@@ -240,6 +240,24 @@ export class AsyncLanguageService {
       ),
     );
 
+  public readonly convertFormattingOptions = (options: monaco.languages.FormattingOptions): ts.FormatCodeOptions => ({
```

Comment:
nit: this doesn't need to be a method on the class, it doesn't use any of the class properties. Make it a function.